functions/Get-DbaClusterSqlInstance.ps1

```diff
@@ -0,0 +1,84 @@
+function Get-DbaClusterSqlInstance
```

Comment:
Works for me

Yes, having a look at Get-DbaClusterActiveNode was on my list of things to do. And that looks prime for splitting out to make scaling up the number of nodes easier to do.